UE/Application/SmsDb.cpp

```diff
@@ -0,0 +1,37 @@
+#include "SmsDb.hpp"
+#include "Sms.hpp"
+#include <vector>
+
+namespace ue
+{
+SmsDb::SmsDb(){
+    std::vector<Sms> vec;
+    //init vector for debug purpose
+    std::string s1 = "Alamakota";
+    this->v_sms = vec;
+    const common::PhoneNumber number{11};
+    v_sms.push_back(Sms(number,s1));
+}
+SmsDb::SmsDb(std::vector<Sms> v_sms){
+    this->v_sms = v_sms;
+}
+
+
+    std::vector<Sms> SmsDb::getAll(){
+        return v_sms;
+    }
+    Sms* SmsDb::getOne(int id){
+
+        return &v_sms.at(id);
+    }
+    void SmsDb::addOne(Sms sms){
+
+        v_sms.push_back(sms);
+    }
+
+    void SmsDb::remove(){
+        v_sms.clear();
+    }
```

Comment:
Whitespace fix (suggestion below). And comments - I think _debug purpose_ in production code should be in unit tests or in logs only, but I am open to discussion.
Also, I would think about the usage of smart pointers in `SmsDb::getOne`.
```suggestion
SmsDb::SmsDb()
{
    std::vector<Sms> vec;
    //init vector for debug purpose
    std::string s1 = "Alamakota";
    this->v_sms = vec;
    const common::PhoneNumber number{11};
    v_sms.push_back(Sms(number, s1));
}

SmsDb::SmsDb(std::vector<Sms> v_sms)
{
    this->v_sms = v_sms;
}

std::vector<Sms> SmsDb::getAll()
{
    return v_sms;
}

Sms* SmsDb::getOne(int id)
{
    return &v_sms.at(id);
}

void SmsDb::addOne(Sms sms)
{
    v_sms.push_back(sms);
}

void SmsDb::remove()
{
    v_sms.clear();
}
```